webpack-server.js

```diff
@@ -0,0 +1,21 @@
+var webpack = require('webpack')
+var WebpackDevServer = require('webpack-dev-server')
+var config = require('./webpack.config')
+
+new WebpackDevServer(webpack(config), {
+  publicPath: config.output.publicPath,
+  hot: true,
+  inline: true,
+  stats: { colors: true },
+  historyApiFallback: true,
+  watchOptions: {
+    aggregateTimeout: 300,
+    poll: 1000
+  },
+}).listen(3000, '0.0.0.0', function (err, result) {
```

Comment:
It should still work with vagrant port forwarding provided it is set up to forward port 3000 to host port 3000.

As for my comment I was just pointing out that having a service listen on `0.0.0.0` will annoy some people :-)